docs/source/users/memory-game.rst

```diff
@@ -0,0 +1,61 @@
+Memory game
```

Comment:
Make this "Memory Task", as that's what it's called in the front end now. Apart from saying something like "The memory task implements a simple memory card game" in the description, I'd also refer to it as "memory task" consistently in the text below.